R/reportLCOE.R

```diff
@@ -138,9 +141,33 @@ reportLCOE <- function(gdx, output.type = "both") {
     pm_SEPrice <- readGDX(gdx, "pm_SEPrice", restore_zeros = FALSE)
 
     ## variables
-    vm_costInvTeDir <- readGDX(gdx, name = c("vm_costInvTeDir", "v_costInvTeDir", "v_directteinv"), field = "l", format = "first_found")[, ttot, ] ## Total direct Investment Cost in Timestep
-    vm_costInvTeAdj <- readGDX(gdx, name = c("vm_costInvTeAdj", "v_costInvTeAdj"), field = "l", format = "first_found")[, ttot, ] ## total adjustment cost in period
-    vm_deltaCap   <- readGDX(gdx, name = c("vm_deltaCap"), field = "l", format = "first_found")[, ttot, ]
+
+
+    ## Total direct Investment Cost in Timestep
+    vm_costInvTeDir <- readGDX(gdx, name = c("vm_costInvTeDir", "v_costInvTeDir", "v_directteinv"), field = "l", format = "first_found")[, ttot, ]
+
+    ## total adjustment cost in period
+    vm_costInvTeAdj <- readGDX(gdx, name = c("vm_costInvTeAdj", "v_costInvTeAdj"), field = "l", format = "first_found")[, ttot, ]
+
+    # capacity additions per year
+    vm_deltaCap <- readGDX(gdx, name = c("vm_deltaCap"), field = "l", format = "first_found")[, ttot, ]
+
```

Comment:
I would add a comment here to explain the next lines, something like: 

apply "modifyInvestmentVariables" to shift from the model-internal time coverage (deltacap and investment variables for step t represent the average of the years from t-4years to t) to the general convention for the reporting template (all variables represent the average of the years from   t-2.5years to t+2.5years)